README.md

```diff
@@ -100,23 +100,53 @@ Django Commons packages.
 
 ## New Project Playbook
 
+Goal: Publish a new release of the project from django-commons repo to PyPI and Test PyPI, using the release workflow.
+
 Assuming the repository name is `repo-name`:
 
+```terraform
+"your-repo-here" = {
+  description = "Your project's description here"
+  allow_merge_commit = true # Allow merge commits on pull requests
+  allow_rebase_merge = true # Allow rebase and merge commits on pull requests
+  allow_squash_merge = true # Allow squash and merge commits on pull requests - Recommended
+  allow_update_branch = true # Allow updating source branch on pull requests
+  has_discussions = false # Enable discussions in project's repository
+  has_wiki = false # Enable wiki in project's repository
+  admins = [
+    # Include people who can release new versions
+    "your-username-here",
+  ]
+  committers = [
+    # Include people who can commit to main / merge changes
+  ]
+  members = [
+    # Include people who can assign/triage tickets
+  ]
+}
+```
+
 ### Pre Transfer Steps
 
 - [ ] Check if the repository meets [inbound requirements][3].
```

Comment:
I would add to the pre-transfer steps to create the 'release workflows' PR in advance. I think it is something that can be done outside of a video call.

Reply:
I like this.

Do you think we should create the PR or should we ask the project owner to create it prior to the meeting? (It should not be merged before the actual transfer for sure).

Reply:
maybe it can just be something like

> A PR to add the release will be necessary. This can be done either by the repo owner OR the Django commons org admins, but should be done prior to the video call. The decision is up to the repo owner

> The PR should NOT be merged before the video call

I am curious as to why it can’t me merged before the migration though 🤔

Reply:
Well, to be precise, it should not run before registering it in pypi with the relevant repo owner/name. Not merging it guarantees it won’t run.

The workflow needs to be registered in pypi with the owner/repo-name allowed to run it - and the owner is changed from the previous one to django-common as part of the transfer.